test-suite/output/Qf_stdlib.out

```diff
@@ -0,0 +1,28 @@
+File "./output/Qf_stdlib.v", line 8, characters 15-30:
+Warning: ssr.Coq.ssrbool has been replaced by Stdlib.Coq.ssrbool.
```

Comment:
Is that really the expected result? Looks like some rev went wrong somewhere.

Reply:
Yes, the bug was introduced in the refactoring, I had checked this manually but I guess it is easy to not to spot it.